spec/beaker-answers/beaker-answers_spec.rb

```diff
@@ -24,548 +26,564 @@
-  end
-
-  it 'should add answers to the host objects' do
+    expect(@answers['vm3']).to include :q_classifier_database_user => 'DFGhjlkj'
+    expect(@answers['vm3']).to include :q_classifier_database_name => 'pe-classifier'
+    expect(@answers['vm3']).to include :q_classifier_database_password => "'~!@\#$%^*-/ aZ'"
+    expect(@answers['vm3']).to include :q_activity_database_user => 'adsfglkj'
+    expect(@answers['vm3']).to include :q_activity_database_name => 'pe-activity'
+    expect(@answers['vm3']).to include :q_activity_database_password => "'~!@\#$%^*-/ aZ'"
+    expect(@answers['vm3']).to include :q_rbac_database_user => 'RbhNBklm'
+    expect(@answers['vm3']).to include :q_rbac_database_name => 'pe-rbac'
+    expect(@answers['vm3']).to include :q_rbac_database_password => "'~!@\#$%^*-/ aZ'"
+  end
+
+  it 'adds answers to the host objects' do
     @ver = '3.4'
     answers = @answers
     hosts.each do |host|
       puts "checking #{host.name}"
-      expect( host[:answers] ).to be === answers[host.name]
+      expect(host[:answers]).to be === answers[host.name]
     end
   end
 
-  it 'should add answers to the host objects' do
+  it 'adds answers to the host objects' do
     hosts.each do |host|
-      expect( host[:answers] ).to be === @answers[host.name]
+      expect(host[:answers]).to be === @answers[host.name]
     end
   end
 
   it 'appends custom answers to generated answers' do
-    expect( hosts[0][:custom_answers] ).to be == { :q_custom => 'LOOKYHERE' }
-    expect( @answers['vm1'] ).to include :q_custom
-    expect( hosts[0][:answers] ).to include :q_custom
+    expect(hosts[0][:custom_answers]).to be == { :q_custom => 'LOOKYHERE' }
+    expect(@answers['vm1']).to include :q_custom
+    expect(hosts[0][:answers]).to include :q_custom
   end
-
 end
 
 describe BeakerAnswers::Version38 do
-  let( :options )     { StringifyHash.new }
-  let( :basic_hosts ) { make_hosts( {'pe_ver' => @ver } ) }
-  let( :hosts ) { basic_hosts[0]['roles'] = ['master', 'agent']
-                  basic_hosts[1]['roles'] = ['dashboard', 'database', 'agent']
-                  basic_hosts[2]['roles'] = ['agent']
-                  basic_hosts[2]['platform'] = 'windows2008r2'
-                  basic_hosts }
-  let( :answers )     { BeakerAnswers::Answers.create(@ver, hosts, options) }
-  let( :upgrade_answers )     { BeakerAnswers::Answers.create(@ver, hosts, options.merge( {:type => :upgrade}) ) }
-
-  before :each do
+  let(:options)     { StringifyHash.new }
+  let(:basic_hosts) { make_hosts('pe_ver' => @ver) }
+  let(:hosts) do
+    basic_hosts[0]['roles'] = %w[master agent]
+    basic_hosts[1]['roles'] = %w[dashboard database agent]
+    basic_hosts[2]['roles'] = ['agent']
+    basic_hosts[2]['platform'] = 'windows2008r2'
+    basic_hosts
+  end
+  let(:answers) { BeakerAnswers::Answers.create(@ver, hosts, options) }
+  let(:upgrade_answers) { BeakerAnswers::Answers.create(@ver, hosts, options.merge(:type => :upgrade)) }
+
+  before do
```

Comment:
Yes, a before defaults to :each, but let's leave it explicit.

Reply:
This cop is called [RSpec/HookArgument](http://www.rubydoc.info/gems/rubocop-rspec/1.7.0/RuboCop/Cop/RSpec/HookArgument), I'm setting it to `EnforcedStyle: each`